spec/core/order.fmf

```diff
@@ -14,3 +17,26 @@ example: |
 link:
   - implemented-by: /tmt/base.py
   - verified-by: /tests/core/order
+
+/predefined-order:
+    title: Predefined order values
+
+    summary: Predefined order values - summary
+
+    description: |
+        30
+            Installation of essential plugin and check requirements.
+
+        50
+            The default order of any object.
+
+        60
+            Application of patches by ``rpm-build`` command invoked by
+            :ref:`discover/fmf</plugins/discover/fmf>` plugin when
+            ``dist-git-source`` key is used.
+
+        70
+            Installation of packages :ref:`required</spec/tests/require>` by tests.
+
+        75
+            Installation of packages :ref:`recommended</spec/tests/recommend>` by tests.
```

Comment:
All these values, except for default `50`, are related to the `prepare` step only. Would it make sense to move them closer to the `prepare` steps docs? And link them from here?

Reply:
I tried, but given how "plan" specification is composed from many stories, I was unable to position this piece correctly under the `prepare` section. Even with `order` it migrated to the beginning or the end of the whole page. I can make it a "plans" topic, but I'm not able to make it a "prepare" topic.

Reply:
What about including it in the `prepare` step description? They are already partially covered there:

https://github.com/teemtee/tmt/blob/686d3bc0161e70507124efb152e9b33a46e8a7a4/spec/plans/prepare.fmf#L15-L22

Reply:
Moved in 8fc3b73c

Reply:
@psss @happz is this resolved?

Reply:
Yes, this has been moved.